Source/WebKit/Platform/spi/ios/RunningBoardServicesSPI.h

```diff
@@ -52,6 +52,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface RBSTarget : NSObject
 + (RBSTarget *)targetWithPid:(pid_t)pid;
++ (RBSTarget *)targetWithPid:(pid_t)pid environmentIdentifier:(NSString *)environment;
```

Comment:
Should we ifdef this behind HAVE_NON_HOSTING_VISIBILITY_PROPAGATION_VIEW?

Reply:
I think this method has been available for a while, and is not directly tied to HAVE_NON_HOSTING_VISIBILITY_PROPAGATION_VIEW, so this may not be needed.